web/pages/api/transaction/info.ts

```diff
@@ -0,0 +1,22 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { getTransactionData } from '../../../common/bitcoin/electrum-api';
```

Comment:
@nieldeckx 
if you merge `master` in this branch and you add the `common` directory to `tsconfig.json` (paths array), you can just refer to this as `@/common/` instead of trying to get the whole subdirectory right